File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.py
def num_func(strs) :
    strs=list(str(strs))
    ans=True
    print(strs)
    num=['0','1','2','3','4','5','6','7','8','9']
    for i in strs :
        if i in num :
           continue
        else :
            ans=False
    return ans
        
def solution(s):
    if len(s)==4 and num_func(s)==True :
        answer = True
    elif  len(s)==6 and num_func(s)==True :
        answer = True
    else :
        answer = False
    return answer
    

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        boolean answer = true;
        
        if (s.length() == 4 || s.length() == 6){
            answer = true;
        }else{
            answer = false;
        }
        
        
        if(answer == true){
            for(int i = 0; i<s.length(); i++){
                if (s.charAt(i) - '0' <0 || s.charAt(i)-'0' > 9){
                    answer = false;
                }
            }
        }
        
        return answer;
    }
}